components/sensor/npi19.rst

```diff
@@ -0,0 +1,124 @@
+NPI-19 Pressure Sensor
+===========================================
+
+.. seo::
+    :description: Instructions for setting up NPI-19 pressure sensors with ESPHome
+    :image: npi19.jpg
+    :keywords: NPI-19 NPI19
+
+The ``npi19`` sensor platform allows you to use your NPI-19 (`datasheet <https://www.amphenol-sensors.com/hubfs/AAS-920-699F-NovaSensor-NPI-19-I2C-061322-web.pdf>`__,
+`product page <https://www.amphenol-sensors.com/en/novasensor/pressure-sensors/3358-npi-19-i2c>`__) pressure sensors with ESPHome.
+
+.. figure:: images/npi19.jpg
+    :align: center
+    :width: 50.0%
+
+    NPI-19 Pressure Sensor.
+
+
+Configuration
+-------------
+:ref:`I²C <i2c>` bus is required to be set up in your configuration for this sensor to work.
+
+.. code-block:: yaml
+
+    # Example configuration entry
+    i2c:
+
+    sensor:
+      - platform: npi19
+        raw_pressure:
+          name: "Raw Pressure"
+          id: raw_pressure
+
+        temperature:
+          name: Temperature
+
+      - platform: copy
+        source_id: raw_pressure
+        name: Pressure
+        state_class: measurement
+        device_class: pressure
+        unit_of_measurement: psi
+
+        filters:
+          - calibrate_linear:
+            - 1638 -> 0.5
+            - 14746 -> 4.5
```

Comment:
Dont need extra info in the example. If people read the page properly they will see the linear calibration suggested below. If you want, you can include the `copy` sensor in the calibration section

```suggestion
    # Example configuration entry
    sensor:
      - platform: npi19
        raw_pressure:
          name: "Raw Pressure"
        temperature:
          name: Temperature
```